src/other/Table.js

```diff
@@ -671,20 +672,36 @@
         var tableMarginHeight = this.fixedHeader() ? this.thead.property("offsetHeight") : 0;
         var totalWidth = 1;
         var tdWidths = {};
+        if(!context.multilineheader()){
+            tds.each(function(d, i) {
+                tdWidths[i] = this.offsetWidth;
+            });
+        }else{
+            tds.each(function(d, i) {
+                tdWidths[i] = "100px";
```

Comment:
I think if the content width is wider than this, then the wider size is used?  If so this is fine.